### ISSUE TITLE
Exceptions on focus changed

### ISSUE THREAD

**Issue by breineng** [github-beta/unity-preview#11]
Version of Unity: 5.6
Version of GitHub for Unity: 0.8-alpha
Platform: Windows

What happened
On focus window canged, give exceptions

Exception(72 times):
```NullReferenceException: Object reference not set to an instance of an object
GitHub.Unity.Logging.GetLogger (System.String context) (at /Users/stanleygoldman/Documents/UnityInternal/src/GitHub.Logging/Logging.cs:63)
GitHub.Unity.Logging.GetLogger (System.Type type) (at /Users/stanleygoldman/Documents/UnityInternal/src/GitHub.Logging/Logging.cs:58)
GitHub.Unity.Logging.GetLogger[ProjectWindowInterface] () (at /Users/stanleygoldman/Documents/UnityInternal/src/GitHub.Logging/Logging.cs:53)
GitHub.Unity.ProjectWindowInterface..cctor () (at /Users/stanleygoldman/Documents/UnityInternal/src/UnityExtension/Assets/Editor/GitHub.Unity/UI/ProjectWindowInterface.cs:19)
Rethrow as TypeInitializationException: An exception was thrown by the type initializer for GitHub.Unity.ProjectWindowInterface
System.Reflection.MonoCMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MonoMethod.cs:513)
Rethrow as TargetInvocationException: Exception has been thrown by the target of an invocation.
System.Reflection.MonoCMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MonoMethod.cs:519)
System.Reflection.MonoCMethod.Invoke (BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MonoMethod.cs:528)
System.Reflection.ConstructorInfo.Invoke (System.Object[] parameters) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/ConstructorInfo.cs:77)
System.Activator.CreateInstance (System.Type type, Boolean nonPublic) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System/Activator.cs:372)
System.Activator.CreateInstance (System.Type type) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System/Activator.cs:254)
UnityEditor.AssetPostprocessingInternal.GetAudioProcessorVersions () (at C:/buildslave/unity/build/Editor/Mono/AssetPostprocessor.cs:339)
UnityEditor.AssetDatabase:Refresh()
GooglePlayGames.Editor.GPGSUpgrader:.cctor() (at Assets/GooglePlayGames/Editor/GPGSUpgrader.cs:110)
UnityEditor.EditorAssemblies:ProcessInitializeOnLoadAttributes()
```

**Comment by StanleyGoldman** [comment]
Yowzer!

@Temka193 thanks for testing and the bug report.
As soon as I know more, I will update you.

P.S.
I'm gonna edit your original post a bit so i can see the stack trace better

**Comment by StanleyGoldman** [comment]
Hey @Temka193 that stack trace is blowing my mind.
Can you enable trace logging in the settings tab and send me the log file?

**Comment by StanleyGoldman** [comment]
Resolved in v0.10-alpha
